src/adv.py

```diff
@@ -111,16 +118,16 @@ def nav():
         print('''
     You do not see a way forward in that direction
             ''')
```

Comment:
watch your indentations here as more white spaces can increase in file space, keep it moderate and clean and you should be fine :)